test/sequential/test-regress-net-response_size.js

```diff
@@ -21,6 +21,11 @@
 
 'use strict';
 const common = require('../common');
+
+// Make sure the net module's server doesn't throw an error when handling
```

Comment:
Can you use hyphens instead of underscores in the file name for consistency? Also for camel cases I think they can either stay in the name as `camelCase` or make them snake cases like `camel-case`